client/core/types.go

```diff
@@ -253,9 +253,7 @@ func matchFromMetaMatchWithConfs(ord order.Order, metaMatch *db.MetaMatch, swapC
 	}
 	if len(swapCoin) > 0 {
 		swap = NewCoin(fromID, swapCoin)
-		if (side == order.Taker && status == order.TakerSwapCast) ||
```

Comment:
Is there a reason for not sending swap information if not on this condition? 

On this issue, after sending the first swap as a maker, the status will be TakerSwapCast and the information will not arrive at the site.

Reply:
> On this issue, after sending the first swap as a maker, the status will be TakerSwapCast and the information will not arrive at the site.

It would be MakerSwapCast.

I always see the conf count update, so I'm confused.

Reply:
After the first maker swap, the order status is not changed to TakerSwapCast?

I am trying to find a reasonable way to reproduce the conf update bug

Reply:
nvm, probably some state I got into. Wasn't able to reproduce the issue on master